### PR TITLE
#2226 tidebit first draft

### PR DESCRIPTION
**Pull request by jjhesk**
adding tidebit first draft. #2226

### CODE REVIEW

File: js/tidebit.js [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+                        'v2/k', // V2KJson
+                        'v2/depth', // V2DepthJson
+                    ],
+                    'post': [],
+                },
+                'private': {
+                    'get': [
+                        'v2/deposits', // V2DepositsJson
+                        'v2/deposit_address', // V2DepositAddressJson
+                        'v2/deposit', //  V2DepositJson
```
Comment (kroitor) [inline js/tidebit.js]: By PEP8 syntax standard Python wants this comment to look like `// V2DepositJson` (one space after the double slash, in your code there's two spaces, Python PEP8 is very critical to that, so it complains about this line and the next one. Everything else looks very clean! Thanks so much!

Comment (kroitor) [reply js/tidebit.js]: @jjhesk can you plz remove the excessive spaces from those two lines? thx!

Comment (jjhesk) [reply js/tidebit.js]: ok @kroitor

**Comment by calvinchankf** [comment]
nice to see this PR

File: js/tidebit.js (hunk 1 above)
Comment (jjhesk) [reply js/tidebit.js]: @kroitor do i have something else need to work on? those two lines spaces are removed.

Comment (kroitor) [reply js/tidebit.js]: @jjhesk nope, everything else is ok, I'll do the cleanup and will merge it shortly, no worries. Thanks so much for your effort!

Comment (jjhesk) [reply js/tidebit.js]: im excited to see it on production!

**Pull request merged**